paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/source/operator/ReadOperator.java

```diff
@@ -94,6 +98,8 @@ public void processElement(StreamRecord<Split> record) throws Exception {
                         .earliestFileCreationEpochMillis()
                         .orElse(FileStoreSourceReaderMetrics.UNDEFINED);
         sourceReaderMetrics.recordSnapshotUpdate(eventTime);
+        // update idleStartTime when reading a new split
+        idleStartTime = FileStoreSourceReaderMetrics.UNDEFINED;
```

Comment:
See `InternalSourceReaderMetricGroup`, it should be `ACTIVE`? Can you just copy the codes?

Reply:
> See `InternalSourceReaderMetricGroup`, it should be `ACTIVE`? Can you just copy the codes?

fixed